runner/webserver.js

```diff
@@ -139,6 +140,9 @@ module.exports = function(wct) {
           log:     wct.emit.bind(wct, 'log:debug'),
         }));
 
+        app.use('/httpbin', httpbin.httpbin);
+        console.log('installed httpbin');
```

Comment:
Just removed the line, I was trying to debug whether this line was being reached and whether `npm link` had worked like I expected it to.